brought company story up to date (#9650)
PostHog/posthog.com: contents/handbook/company/story.md

We've decided to double down on mass adoption of the platform in high potential startups instead of focusing on enterprise. Simply, this will better help us increase the number of successful products in the world. As a result, we've removed support for paid self-hosted deployment and are doubling down on our open source and cloud projects. We have released a PostHog free tier.

> ## January 2024: well, that was good

Reviewer: ```suggestion
## January 2024: Well, that was good
```

> That was quite the year We wound up quadrupling our revenue but only increasing our net headcount by three people in 2023. Last year, we validated that we could get multiple products to product market fit.

Reviewer: ```suggestion
That was quite the year.

We wound up quadrupling our revenue, but only increasing our net headcount by three people in 2023. Last year, we validated that we could get multiple products to product-market fit.
```

> ## April 2024: we're now the standard for startups

Reviewer: ```suggestion
## April 2024: We're now the default for startups
```

> 54% of the first YCombinator batch this year adopted PostHog. Tim and James turned up to talk at batch events and we were surprised at the number of groupies wearing PostHog merch (which is by now really cool - we've gone way beyond the logo-on-a-black-t-shirt standard). We're in the top three products as far as we can tell used by YC companies.

Reviewer: ```suggestion
54% of the first YCombinator batch this year adopted PostHog. 

Tim and James turned up to talk at batch events and we were surprised at the number of groupies wearing [PostHog merch](/merch) – our merch is really cool now, e've gone way beyond the logo-on-a-black-t-shirt standard. 

As far as we can tell, we're in the top three products used by YC companies.
```

> ## July 2024: price cuts ftw

Reviewer: ```suggestion
## July 2024: Price cuts ftw
```

We cut pricing by up to 90% for our two most popular products, including for our existing customer base. This was [popular with users](https://x.com/posthog/status/1815796915814179137) and led to faster growth.

> We've started doing growth reviews for almost every product we have. We run through each product's metrics (revenue/usage/support/performance) and feedback / reasons for any churn that has happened, so we can truly treat each small team like a startup. This session is designed so the engineer team leads may choose to reprioritize work, or not.

Reviewer: ```suggestion
We've started doing growth reviews for almost every product we have. We run through each product's metrics (revenue/usage/support/performance) and feedback / reasons for any churn that has happened, so we can truly treat each small team like a startup. This session is designed so the engineering team leads may choose to reprioritize work, or not.
```

## October 2024: 100,000 customers, and speeding up - more products and more people

> We hit 100,000 customers either paying or free, and over a quarter of a million users. We've started hiring a lot faster as growth has continued this year. We're now 65 ish people with ~9 products.

Reviewer: ```suggestion
We hit 100,000 customers either paying or free, and over a quarter of a million users. We've started hiring a lot faster as growth has continued this year. We're now 65 ish people with ~9 products.
```

> We've added some people in sales, but it is strictly (i) sales assist - talking to people that have asked to speak to us and (ii) cross sell to existing customers. We do _not_ do outbound, so we can remain efficient and either hire more engineers or cut our pricing for our customers so more of them recommend us! We've hired a sales engineer super early (Mine, she's awesome) and we're really working on the culture in this team proactively.

Reviewer: ```suggestion
We've added some people in sales, but it is strictly (i) sales assist, talking to people that have asked to speak to us, and (ii) cross sell to existing customers. 

We do _not_ do outbound, so we can remain efficient and either hire more engineers or cut our pricing for our customers so more of them recommend us! 

We've hired a sales engineer super early (Mine, she's awesome) and we're really working on the culture in this team proactively.
```

> Strategy-wise, we're  just leaning into our basic three principles which we're seeing more and more evidence are working well:

Reviewer: ```suggestion
Strategy-wise, we're just leaning into our basic three principles, which we're seeing more and more evidence are working well:
```

> * all the tools in one - we want to go wider still. We think we can provide _every_ piece of SAAS that startups use, starting with those closest to customer data. We want to expand to a customer support product, the marketing and sales stack of tools too.
> * get in first - don't go upmarket. We're closing enterprises regularly, but we're not trying that hard here. We're trying to stay away from complex migrations for users who use many products already.
> * be the source of truth - our own data warehouse is now available and very popular

Reviewer: ```suggestion
1. **All the tools in one** – We want to go wider still. We think we can provide _every_ piece of SaaS that startups use, starting with those closest to customer data. We want to expand to a customer support product, the marketing and sales stack of tools too.

2. **Get in first** – Don't go upmarket. We're closing enterprises regularly, but we're not trying that hard here. We're trying to stay away from complex migrations for users who use many products already.

3. **Be the source of truth** – Our own data warehouse is now available and very popular.
```

> Revenue wise we're in the low $10s of millions of ARR. We're very strongly default alive and will struggle to not end up profitable next year. Every time we get close to being profitable, we start speeding up hiring. Revenue growth is fast enough and we're getting so many unprompted offers for investment (that we aren't taking) that money isn't really a meaningful constraint any more.

Reviewer: ```suggestion
On revenue, we're in the low $10s of millions of ARR. We're very strongly default alive and will struggle to not end up profitable next year. Every time we get close to being profitable, we start speeding up hiring. 

Revenue growth is fast enough and we're getting so many unprompted offers for investment (that we aren't taking) that money isn't really a meaningful constraint any more.
```

> Whilst we have a great grip on each product's individual performance, our understanding of cross sell is a little weak so we're working on that now.

Reviewer: ```suggestion
Whilst we have a great grip on each product's individual performance, our understanding of cross sell is a little weak, so we're working on that now.
```

> Our marketing is getting weirder. It's more and more fun. We've commissioned a puppet, coming in January. Watch this space.

Reviewer: ```suggestion
Our marketing is getting weirder. It's more and more fun. We've commissioned a puppet, coming in January. Watch this space. Our newsletter, [Product for Engineers](/newsletter), now has 20,000 subscribers and it's growing fast.
```

> We're realizing that the more ambitious we are, the easier it gets (customers get excited, investors get excited, employees get excited). We can now see a real path to being a $100bn+ company and changing how software teams work industry wide.

Reviewer: ```suggestion
We're realizing that the more ambitious we are, the easier it gets – customers get excited, investors get excited, employees get excited. We can now see a real path to being a $100bn+ company and changing how software teams work industry-wide.
```

We've added some people in sales, but it is strictly (i) sales assist, talking to people that have asked to speak to us, and (ii) cross sell to existing customers. 

We do _not_ do outbound, so we can remain efficient and either hire more engineers or cut our pricing for our customers so more of them recommend us! 

We've hired a sales engineer super early (Mine, she's awesome) and we're really working on the culture in this team proactively.

Strategy-wise, we're just leaning into our basic three principles, which we're seeing more and more evidence are working well:

1. **All the tools in one** – We want to go wider still. We think we can provide _every_ piece of SaaS that startups use, starting with those closest to customer data. We want to expand to a customer support product, the marketing and sales stack of tools too.

2. **Get in first** – Don't go upmarket. We're closing enterprises regularly, but we're not trying that hard here. We're trying to stay away from complex migrations for users who use many products already.

3. **Be the source of truth** – Our own data warehouse is now available and very popular.

> On revenue, we're in the low $10s of millions of ARR. We're very strongly default alive and will struggle to not end up profitable next year. Every time we get close to being profitable, we start speeding up hiring. 

Reviewer: ```suggestion
Revenue is in the low $10s of millions of ARR. We're very strongly default alive and will struggle to not end up profitable next year. Every time we get close to being profitable, we start speeding up hiring. 
```

> Revenue growth is fast enough and we're getting so many unprompted offers for investment (that we aren't taking) that money isn't really a meaningful constraint any more.
> 
> Whilst we have a great grip on each product's individual performance, our understanding of cross sell is a little weak, so we're working on that now.

Reviewer: ```suggestion
Revenue growth is fast enough and we're getting so many unprompted offers for investment (that we aren't taking) that money isn't really a meaningful constraint any more. Whilst we have a great grip on each product's individual performance, our understanding of cross sell is a little weak, so we're working on that now.
```
